TimberAPI/Dependency/IDependency.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Bindito.Core;
+using TimberbornAPI.Common;
+
+namespace TimberbornAPI.DependencySystem
+{
+    public interface IDependency
+    {
+        /// <summary>
```

Comment:
Same doc comment in interface and implementation is gonna be tough to maintain when something changes. Would be enough to maybe have it only on the interface?